packages/caching-minifier/caching-compiler.js

```diff
@@ -0,0 +1,200 @@
+// Copied from https://raw.githubusercontent.com/meteor/meteor/devel/packages/caching-compiler/caching-compiler.js
```

Comment:
This file was copied from the caching-compiler core package. It probably would be good to either import the file from there, or move the caching minifier class into that package.

I didn't do either because I was trying to avoid my minifier from having a dependency on the ecmascript package, since it added 45mb.

Reply:
Good insight!